modules/install/pages/server-processes.adoc

```diff
@@ -142,3 +142,121 @@ Instance count is configured in UI.
 | Analytics
 | _/opt/couchbase/bin/_
 |===
+
+== Windows
+
+The following table lists the Couchbase processes that run on the Windows platforms.
+
+.Couchbase Processes on Linux
+[#table-processes-windows,cols="1,2,1,2",options="header"]
+|===
+
+| Process | Description | Service | Path
+
+| `backup.exe`
+| Backup application for Couchbase data
+| Backup
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `cbas.exe`
+| Go-wrapper that communicates with `ns_server` and manages the lifecycle of the Analytics Driver
+| Analytics
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `cbcollect_info.exe`
+| Utility used to collect Couchbase server logs (will be seen only during log collection)
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `cbft.exe`
+| Couchbase Full-Text Search (FTS) service
+| Search
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `cbq-engine.exe`
+| Couchbase Query service
+| Query
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `epmd.exe`
+| Erlang-specific process which acts as a name server for Erlang distribution
+| Core
+| _C:\Program Files\Couchbase\Server\erts-x.x.x.x\bin_
+
+| `erl.exe`
+| Erlang process used by the name server.
+| Core
+| _C:\Program Files\Couchbase\Server\erts-x.x.x.x\bin_
+
+| `eventing-consumer.exe` 
+| Eventing worker (multiple instances per node).
+Instance count is configured in UI.
+| Eventing
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `eventing-producer.exe` 
+| Eventing supervisor service (one instance per node)
+| Eventing
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `goport.exe` 
+| Process that acts as a bridge between `ns_server` (Erlang) and the other server components (`cbq- engine.exe`, `cbft.exe`, etc.)
+| Query
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `gosecrets.exe`
+| Service that is used to encrypt the cluster configuration stored on disk
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `goxdcr.exe`
+| Cross Data Center Replication (XDCR) - replicates data from one cluster to another
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `indexer.exe`
+| Index service
+| Index
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `godu.exe` 
+| Utility in Go to get disk usage stats
+| Core
+| _C:\Program Files\Couchbase\Server\bin\priv_
+
+| `inet_gethost.exe`
+| Built-in Erlang port process that is used to perform name service lookup
+| Core
+| _C:\Program Files\Couchbase\Server\erts-x.x.x.x\bin_
+
+
+| `java.exe` (Analytics Driver)
+| JVM running the Analytics NC and CC
+| Analytics
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `memcached.exe`
+| Data service responsible for storing user data
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `projector.exe`
+| Extracts secondary key from documents
+| Core
+| _C:\Program Files\Couchbase\Server\bin_
+
+| `prometheus.exe`
+| Metrics engine used by Analytics
```

Comment:
AFAIK this isn't specific to analytics 

Metrics engine used by Couchbase to store all stats
This is owned by ns_serv I believe.

Reply:
Fixed